packages/mermaid/src/diagrams/xychart/parser/xychart.jison

```diff
@@ -0,0 +1,201 @@
+%lex
+%options case-insensitive
+
+%x string
+%x md_string
+%x title
+%x open_directive
+%x type_directive
+%x arg_directive
+%x close_directive
+%x acc_title
+%x acc_descr
+%x acc_descr_multiline
+%s axis_data
+%s data
+%s data_inner
+%%
+\%\%\{                                    { this.begin('open_directive'); return 'open_directive'; }
+<open_directive>((?:(?!\}\%\%)[^:.])*)    { this.begin('type_directive'); return 'type_directive'; }
+<type_directive>":"                       { this.popState(); this.begin('arg_directive'); return ':'; }
+<type_directive,arg_directive>\}\%\%      { this.popState(); this.popState(); return 'close_directive'; }
+<arg_directive>((?:(?!\}\%\%).|\n)*)      return 'arg_directive';
+\%\%(?!\{)[^\n]*                          /* skip comments */
+[^\}]\%\%[^\n]*                           /* skip comments */
```

Comment:
This only supports one character before `%%`. Is that on purpose or was the intention to comment out any line that contains `%%`?

Reply:
@ibrahimWassouf I have not checked, this must be a plain copy-paste from other grammars

Reply:
Yes I have copied it from a different file, I don't know exactly why it is there